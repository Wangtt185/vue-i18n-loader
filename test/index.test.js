import { bundleAndRun } from './utils'

test('basic', async () => {
  const { module } = await bundleAndRun('basic.vue')
  expect(module.__i18n).toMatchSnapshot()
})

test('special characters', async () => {
  const { module } = await bundleAndRun('special-char.vue')
  expect(module.__i18n).toMatchSnapshot()
})

test('multiple', async () => {
  const { module } = await bundleAndRun('multiple.vue')
  expect(module.__i18n).toMatchSnapshot()
})

test('import', async () => {
  const { module } = await bundleAndRun('import.vue')
  expect(module.__i18n).toMatchSnapshot()
})
