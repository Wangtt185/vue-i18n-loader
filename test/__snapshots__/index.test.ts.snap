// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`basic 1`] = `
Array [
  "{\\"en\\":{\\"hello\\":\\"hello world!\\"}}",
]
`;

exports[`import 1`] = `
Array [
  "{\\"en\\":{\\"hello\\":\\"hello world!\\"}}",
]
`;

exports[`multiple 1`] = `
Array [
  "{\\"en\\":{\\"hello\\":\\"hello world!\\"}}",
  "{\\"ja\\":{\\"hello\\":\\"こんにちは、世界！\\"}}",
]
`;

exports[`special characters 1`] = `
Array [
  "{\\"en\\":{\\"hello\\":\\"hello\\\\ngreat\\\\t\\\\\\"world\\\\\\"\\"}}",
]
`;
